src/main/java/mt/edu/um/cf2/jgribx/grib2/Grib2Record.java

```diff
@@ -31,11 +31,11 @@
     protected GribRecordIS is;
     protected Grib2RecordIDS ids;
     protected List<Grib2RecordLUS> lusList = new ArrayList();
-    protected List<Grib2RecordGDS> gdsList = new ArrayList();
+    public List<Grib2RecordGDS> gdsList = new ArrayList();
```

Comment:
Why do you change this to public?

Reply:
I wanted to be able to iterate over the data.

Reply:
Hmm how about adding getter methods in the same file instead?

Reply:
Sure :)